RELEASE_NOTES.txt

```diff
@@ -33,9 +33,7 @@ Apache Ignite Spring Data Modules 1.0
 * Added the ability to connect to the Ignite cluster through the Ignite thin client for the Spring Data extensions
 * Fixed closing of Spring Data Ignite resources after destroying the Spring context if they were automatically started by the manager
 
-The following integrations were migrated to the Apache Ignite Extensions repository:
```

Comment:
Already released notes should not be changed

Reply:
Reverted this change